web/templates/help_menu.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="navbar-dropdown-menu" id="help-menu-dropdown" aria-labelledby="help-menu" data-simplebar>
+<div class="navbar-dropdown-menu" id="help-menu-dro" aria-labelledby="help-menu" data-simplebar>
```

Comment:
this `-dro` looks like a typo?

Reply:
Sorry. I clearly recall fixing this before opening the PR.. 👻